src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ *  Deletes tag(s) in the address book.
+ */
+public class DeleteTagCommand extends TagCommandGroup {
+
+    public static final String COMMAND_SPECIFIER = "delete";
+    public static final String COMMAND_WORD = COMMAND_GROUP + " " + COMMAND_SPECIFIER;
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes tags with the names given. "
+            + "Parameters: "
+            + "[TAG1] [TAG2] ... \n"
+            + "Example: " + COMMAND_WORD + " tag1 tag2 tag3";
+
+    public static final String MESSAGE_SUCCESS = "Tags deleted: %1$s";
+    public static final String MESSAGE_DUPLICATE_TAGS = "Tag(s) %1$s does not exist in Rapportbook!";
```

Comment:
I think `MESSAGE_TAGS_DONT_EXIST` would be a more suitable name

Reply:
you're right. i forgot to change it after copy-pasting